jsonextra/json_extra.py

```diff
@@ -22,6 +22,10 @@ def disable_rex(rex):
     globals()[rex] = None
 
 
+def enable_date_rex():
```

Comment:
This only addresses date regex. It would better be one, which could work with other regex patterns too.

<wishlist>A better idea would have a manager class, which could handle what is enabled or disabled. Also, a nice touch would be to have a context decorator, which could enable/disable regexes within a context block.</wishlist>

Reply:
Thanks for your comment. I've introduced a `regex_manager` to manage the enabling and disabling of regexs in the latest commit.

Reply:
That is pretty similar to what I had in mind. There are a few things that can be done, I will leave comments on the PR.